src/pretix/eventyay_common/navigation.py

```diff
@@ -105,6 +105,23 @@ def get_event_navigation(request: HttpRequest, event: Event) -> List[MenuItem]:
     # Merge plugin items into default navigation
     merge_in(nav, sorted_plugin_items)
 
+    # Filter out unwanted menu items if their URL contains one of these paths
```

Comment:
So, basically this filters the currently installed plugins. As a matter for fact we dont want any plugin to show up in future.

Reply:
Yes, sounds good.